app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ContactDetailsTelephoneNumberController.scala

```diff
@@ -85,10 +86,10 @@ class ContactDetailsTelephoneNumberController @Inject() (
                 )
               )
             ),
-          emailAddress =>
-            updateRegistration(emailAddress).map {
+          phoneNumber =>
+            updateRegistration(phoneNumber).map {
               case Right(_) =>
-                Redirect(groupRoutes.OrganisationListController.displayPage())
+                Redirect(groupRoutes.ConfirmContactAddressController.displayPage())
```

Comment:
You might want to consider hooking in the new group member CYA page which I've just merged here.

Reply:
Done